### ISSUE TITLE
Importing may cause de-styling of Bulma components

### ISSUE THREAD

**Issue by groenroos** [apnsngr/bulma-popover#11]
Thanks for the great plugin!

However, I noticed that utilising this plugin via `@import "bulma-popover"` can de-style certain components that rely on Bulma utilities.

Given a stylesheet like this;

```scss
@import "bulma";
@import "bulma-popover";
```

And an HTML file like so;

<details>
  <summary><code>index.html</code></summary>

```html
<html>
<head>
	<link rel="stylesheet" href="/app.css">
</head>
<body>
	<div class="section">
		<div class="container">
			<input type="text" class="input" placeholder="Text input" />
		</div>
	</div>
</body>
</html>
```

</details>

Any text `.input` will lose its default border and box-shadow:

Expected | Actual
-|-
<img width="428" alt="Screenshot 2022-02-22 at 18 00 15" src="https://user-images.githubusercontent.com/951547/155191772-3caadb29-eac1-4605-8497-08527fd8f248.png"> | <img width="429" alt="Screenshot 2022-02-22 at 17 59 54" src="https://user-images.githubusercontent.com/951547/155191763-714f4195-bd44-4f3c-8d2b-0ea18b86dc2c.png">

The `.input` element will only appear correctly if the import order is reversed;

```scss
@import "bulma-popover";
@import "bulma";
```

I _think_ this has got to do with how `bulma-popover.scss` loads all of Bulma's utilities on line 1, which are `@import`ed into the global namespace, and thus overwrite/hoist these definitions over any of the more specific definitions inside of `bulma`.

While reversing the import order is a functioning workaround, I don't think the `bulma-popover` import should be able to pollute Bulma's components like this.

**Comment by apnsngr** [comment]
Thank you so much for reporting this issue! I meant to get to this a little earlier, but work has been extra busy lately.

I believe I have fixed the problem. Do you mind testing it by installing the latest beta version (`npm install bulma-popover@beta`)? After you confirm that the problem is solved I will publish it officially.

**Comment by apnsngr** [comment]
Fixed in release 1.1.1
